platforms/software/testing-base/src/main/java/org/gradle/api/tasks/testing/TestEventReporter.java

```diff
@@ -77,9 +77,9 @@ public interface TestEventReporter extends AutoCloseable {
      *
      * @param logTime the time the metadata was logged, should be between the start and end times of the test (inclusive)
      * @param values the metadata values, containing in a map which must be serializable by the Tooling API
-     * @since 8.13
+     * @since 9.3.0
```

Comment:
🤔 Are we required to change the "since" on these when changing the generic type arg?  If no, I don't think we should.